jung-visualization/src/main/java/edu/uci/ics/jung/visualization/transform/AbstractLensSupport.java

```diff
@@ -146,13 +142,13 @@ public boolean useTransform() {
   }
 
   /** @return the lens */
```

Comment:
update comment please

Reply:
Ha! Intellij renaming refactor is really good, but not perfect, huh?